package.json

```diff
@@ -4,7 +4,8 @@
   "description": "ESLint config following Wikimedia code conventions.",
   "main": ".eslintrc.json",
   "scripts": {
-    "test": "eslint test/fixtures/*.js && node test/index.js"
+    "test": "eslint . && node test/valid.js && node test/invalid.js",
```

Comment:
No, it can. It's intentionally done that way. Any violations are marked with `disable-next-line`. I just forgot one of them. Fixing...